src/modules/articles/components/ArticleMedia.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { Gallery } from '../../media/components';
+import { Lightbox } from '../../core/components';
+import ArticleMedium from './ArticleMedium';
+
+import { IMedium, IArticle } from '../queries';
+
+interface IProps {
+    article: IArticle,
+    media: Array<IMedium>
+}
+
+export const ArticleMedia: React.FunctionComponent<IProps> = ({article, media}) => {
+    const [isLightboxOpen, setIsLightboxOpen] = React.useState(false);
```

Comment:
`setIsLightboxOpen`? Why not `openLightbox`? Or `toggleLightbox`?

Reply:
From what I've seen that's the useState convention: variableName then setVariableName, probably because it mimics setState(). I think it's probably best to stick with the convention.

Reply:
Eh, convention is okay, but if it leads to awkward naming (in general avoid two verbs in a row like `setIs`), then I'd say it's okay to go against it. The prime directive of programming is readability (imo) and that surpasses convention or consistency.